src/System.Net.Security/tests/FunctionalTests/ClientDefaultEncryptionTest.cs

```diff
@@ -43,7 +43,7 @@ public ClientDefaultEncryptionTest()
 
                 using (var sslStream = new SslStream(client.GetStream(), false, AllowAnyServerCertificate, null))
```

Comment:
All modified tests were already enabled. In fact, of all the test files touched in this PR, all the tests in those files are already enabled.